Microsoft.Toolkit.Uwp.UI.Animations/CompositionAnimations/KeyFrames/TypedKeyFrame.cs

```diff
@@ -0,0 +1,38 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations
+{
+    /// <summary>
+    /// Provides type implementation of <see cref="KeyFrame"/>
+    /// </summary>
+    /// <typeparam name="T">The type of property being animated</typeparam>
+    public abstract class TypedKeyFrame<T> : KeyFrame
+    {
+        /// <summary>
+        /// Gets or sets the value at the specific key
+        /// </summary>
+        public T Value
+        {
+            get { return (T)GetValue(ValueProperty); }
+            set { SetValue(ValueProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="Value"/> dependency property
+        /// </summary>
+        public static readonly DependencyProperty ValueProperty =
+            DependencyProperty.Register("Value", typeof(T), typeof(TypedKeyFrame<T>), new PropertyMetadata(0.0));
```

Comment:
`Value` should use `nameof(Value)`
The default value of the dependency property is a double, but the type is T. This would fail if T isn't a double.

Reply:
Good catch, will fix

Reply:
Done